### PR TITLE
[16.01] Handle sections in view workflow

### LINKED ISSUE #1723: Tools with sections fail during "View workflow"
Filing this officially since I mentioned it in chat once and forgot to file properly.

```
  return render_row_for_param(context,param,value,other_values,prefix,step)
File '/galaxy/database/compiled_templates/workflow/display.mako.py', line 233 in render_row_for_param
  __M_writer(filters.html_escape(unicode(param.get_label() )))
AttributeError: 'Section' object has no attribute 'get_label'
```

Galaxy throws up an internal server error when trying to view workflows which use tools which have sections.

### PR DESCRIPTION
**Pull request by hexylena**
Fixes #1723, backported.

ping @guerler

### CODE REVIEW

**Comment by jmchilton** [comment]
:+1:

**Comment by guerler** [comment]
Thanks. :+1:

**Comment by hexylena** [comment]
@nsoranzo would you be willing to review? (since you requested the close on the one on dev, since that would've been merged with aysam's +1, and this requires more momentum)

File: templates/webapps/galaxy/workflow/display.mako [hunk 1]
```diff
@@ -30,6 +30,15 @@
       <% new_prefix = prefix + input.name + "|" %>
       ${row_for_param( input.test_param, group_values[ input.test_param.name ], other_values, prefix, step )}
       ${do_inputs( input.cases[ current_case ].inputs, group_values, new_prefix, step, other_values )}
+    %elif input.type == "section":
+      <% new_prefix = prefix + input.name + "|" %>
+      <% group_values = values[input.name] %>
+      <div class="form-title-row"><b>${input.title}:</b></div>
+      <div class="repeat-group">
+        <div class="repeat-group-item">
```
Comment (nsoranzo) [inline templates/webapps/galaxy/workflow/display.mako:38]: Just a thought: do you need the nested `repeat-group` and `repeat-group-item` divs or `repeat-group-item` would be enough?

Comment (hexylena) [inline templates/webapps/galaxy/workflow/display.mako:38]: @nsoranzo required for CSS to work properly. If someone has made changes to styling on repeats, I would like sections to appear the same. If we don't use an identical HTML structure, there's no guarantee that their selectors will work correctly if they referred to a parent.

**Pull request merged**